lib/liquid/standardfilters.rb

```diff
@@ -177,6 +177,10 @@ def append(input, string)
       input.to_s + string.to_s
     end
 
+    def concat(input, array)
+      InputIterator.new(input).concat(array)
```

Comment:
because if use only input.concat(array) the code will be able to concat strings like assert_equal('1,210', @filters.concat('1,2', '10' ))